<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notifications</class>
 <widget class="QDialog" name="Notifications">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="push_refresh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 92, 0);
color: rgb(46, 52, 54);</string>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_back">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 92, 0);
color: rgb(46, 52, 54);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_approve">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 92, 0);
color: rgb(46, 52, 54);</string>
   </property>
   <property name="text">
    <string>Approve</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_delete">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>370</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 92, 0);
color: rgb(46, 52, 54);</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>451</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Approved</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>461</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Notifications:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>71</y>
     <width>471</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_views">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);
color: rgb(46, 52, 54);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
